exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillDatabaseMetaDataImpl.java

```diff
@@ -1273,5 +1295,11 @@ public boolean generatedKeyAlwaysReturned() throws SQLException {
     return super.generatedKeyAlwaysReturned();
   }
 
+  private ResultSet executeSql(String sql) throws SQLException {
```

Comment:
-1

I don't think executing a query is the right way to figure out server properties. Maybe a set of server properties should be returned as part of establishing a connection (identifier quote string being one such property).